custom/enikshay/ucr/data_sources/test_2b_v2.json

```diff
@@ -0,0 +1,3377 @@
+{
+    "domains": [
+        "enikshay",
+        "sheel-enikshay",
+        "enikshay-reports-qa",
+        "enikshay-performance-test"
+    ],
+    "server_environment": [
+        "enikshay",
+        "softlayer"
```

Comment:
I don't think these need to be added to softlayer anymore. Is there any testing going on in that cluster

Reply:
sorry. removing this.